spec/requests/user_pages_spec.rb

```diff
@@ -56,4 +56,33 @@
 	  it { should have_title(user.name) }
 	end
 
+  describe "edit" do
+    let(:user) { FactoryGirl.create(:user) }
+    before { visit edit_user_path(user) }
+
+    describe "page" do
+      it { should have_content("Update your profile") }
+      it { should have_title("Edit user") }
```

Comment:
[rubocop] Prefer single-quoted strings when you don't need string interpolation or special symbols.